src/test/java/com/wikia/webdriver/common/contentpatterns/AdsFandomContent.java

```diff
@@ -20,6 +23,9 @@
     SLOT_SELECTORS.put(TOP_BOXAD, "div[id$='TOP_BOXAD_0__container__']");
     SLOT_SELECTORS.put(INCONTENT_BOXAD, "div[id$='INCONTENT_BOXAD_0__container__']");
     SLOT_SELECTORS.put(BOTTOM_BOXAD, "div[id$='BOTTOM_BOXAD_0__container__']");
+    SLOT_SELECTORS.put(GPT_TOP_LEADERBOARD, "#gpt-top-leaderboard");
+    SLOT_SELECTORS.put(GPT_BOTTOM_LEADERBOARD, "#gpt-bottom-leaderboard-desktop");
+    SLOT_SELECTORS.put(GPT_BOTTOM_LEADERBOARD_MOBILE, "#gpt-bottom-leaderboard-mobile");
```

Comment:
Tests will not pass if I use only TOP_LEADERBOARD selector (this selector is required to get iframe id), to use methods from VideoFanTakeover class  I'm using GPT_TOP_LEADERBOARD selector